.github/workflows/release-please.yml

```diff
@@ -42,7 +42,7 @@ jobs:
       - name: Publish to NPM
         env:
           NODE_AUTH_TOKEN: ${{secrets.NPM_TOKEN}}
-        run: npm publish --access public
+        run: npm run publish-all
```

Comment:
This script attempts to publish the client and server modules by calling a script in each of them. If the artifact already exists, it's a no-op, similar to what we do in many other contribs.